src/functions.rs

```diff
@@ -474,6 +474,7 @@ expr_fn!(
 );
 expr_fn!(sqrt, num);
 expr_fn!(starts_with, arg1 arg2, "Returns true if string starts with prefix.");
+expr_fn!(ends_with, arg1 arg2, "Returns true if string ends with suffix.");
```

Comment:
We should probably match the argument names to upstream: `string suffix`

(I assume you just matched what `starts_with` has. That was either my error as I made the initial port or the upstream UDF changed)

https://docs.rs/datafusion/latest/datafusion/functions/expr_fn/fn.ends_with.html

Reply:
updated!  Also updated the argument names for starts_with

Reply:
actually...looks like the upstream for starts_with is `arg1`, and `arg2`...should i revert it back?

Reply:
https://docs.rs/datafusion/latest/datafusion/functions/expr_fn/fn.starts_with.html